### ISSUE TITLE
Post-Lemmatisation Stopword Filter

### ISSUE THREAD

**Issue by ericjanto** [ericjanto/lex#20]
TL;DR: check lemmatised token against stopword list.

The current pre-processing step allows for stopwords to slip through the `_is_relevant_token` filter:
https://github.com/ericjanto/lex/blob/9cce5cb3dc309ebc30c51742995c989fd9e509b5/backend/textparser.py#L202

This happens when a pre-lemmatised word, e.g. `saying` is not recognised as stopword. The, after lemmatisation `say` is not checked against the stopword list, leading to `say` being registered as a lemma.

This is especially suboptimal as it leads to incoherent displaying of `say` in context values; in those where `say` appears in its original form, it doesn't link to the lemma.
